ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/diagnostic/DiagnosticMessages.java

```diff
@@ -89,11 +89,11 @@ public enum DiagnosticMessages {
     OAS_CONVERTOR_127("OAS_CONVERTOR_127", "Generated OpenAPI definition does not contain the request" +
             " parameter information from the interceptor pipeline. Cause: %s", DiagnosticSeverity.WARNING),
     OAS_CONVERTOR_128("OAS_CONVERTOR_128", "Given example path `%s` is not exist, therefore " +
-            "generated yaml doesn't contain example for `%s` ", DiagnosticSeverity.WARNING),
+            "generated yaml does not contain example for `%s` ", DiagnosticSeverity.WARNING),
     OAS_CONVERTOR_129("OAS_CONVERTOR_129", "Given example path `%s` should be JSON file for" +
             " example `%s`", DiagnosticSeverity.WARNING),
-    OAS_CONVERTOR_130("OAS_CONVERTOR_130", "Following issue occurred parsing given example:" +
-            " %s", DiagnosticSeverity.WARNING),
+    OAS_CONVERTOR_130("OAS_CONVERTOR_130", "The given JSON value for example contains parsing " +
```

Comment:
How about this error message:
```
Generated OpenAPI definition does not contain the example details since the JSON value provided for the example has parser errors
```